### ISSUE TITLE
_disableAccessibilityState

### ISSUE THREAD

**Issue by danielghost** [adaptlearning/adapt_framework#2772]
It looks like this attribute is missing from the core schemas except for _component.model.schema_.

**adapt-contrib-boxMenu:**
Should `_disableAccessibleState` be changed to `_disableAccessibilityState` for consistency with other models. Also missing from the schema.

**Comment by oliverfoster** [comment]
We're probably going to drop `_disableAccessibleState ``in favour of `_isA11yCompletionDescriptionEnabled`.

`_disabledAccessibleState` is really now `_shouldRenderHeaderViews`.

https://github.com/adaptlearning/adapt_framework/search?q=_disableAccessibilityState&unscoped_q=_disableAccessibilityState

https://github.com/adaptlearning/adapt_framework/search?q=_isA11yCompletionDescriptionEnabled&unscoped_q=_isA11yCompletionDescriptionEnabled

**Comment by moloko** [comment]
partial duplicate of #2331
